### PR TITLE
Adding a file length option and documentation to iterate over packets

### PR DESCRIPTION
**Pull request by shealutton**
Kisom, thanks for writing this lib. It was really helpful for me, but I needed a way to iterate over 20 million packets and pull raw data out of the payload. I added a length option and some documentation on how to read over every packet to the README. Would you incorporate this into your distribution? Thanks, Shea

### CODE REVIEW

**Comment by kisom** [comment]
Hello @shealutton --- this looks really useful! One question: can you add a test for this (and optionally, your name to the AUTHORS file)?

**Comment by shealutton** [comment]
Yes, I will do it next week when I am on vacation @kisom .

**Comment by shealutton** [comment]
@kisom This was my first try at using this style of test code, please take a look and confirm that my test is legit and works the way I expect. ;) I think it is good though. I also incremented the minor version number, please make sure you agree with that change too. Thanks, Shea

File: pcapfile/test/savefile_test.py [hunk 1]
```diff
@@ -106,6 +106,8 @@ def test_basic_import(self):
         self.assertTrue(self.capfile.valid, 'invalid capture file')
         self.assertEqual(len(self.capfile.packets), 23,
                          'wrong number of packets!')
+        self.assertEqual(self.capfile.__length__(), 23,
+                         '__length__ not reporting correct number of packets')
```
Comment (kisom) [inline pcapfile/test/savefile_test.py]: One minor thing, it'd probably be good to also test `len(self.capfile)` to make sure it's behaving correctly.

Comment (shealutton) [inline pcapfile/test/savefile_test.py]: I am not sure what you mean @kisom. I think lines 107 and 108 do that don't they? Can you elaborate?

Comment (kisom) [inline pcapfile/test/savefile_test.py]: @shealutton Lines 107 and 108 test `self.capfile.packets`, not `self.capfile` --- the thinking is make sure they're the same.

**Pull request merged**